test/rest-smoketest.sh

```diff
@@ -200,13 +212,25 @@ dotest "204::*::*" "${obj1_vers1}" -X DELETE
 dotest "404::*::*" "${obj1_vers1}"
 dotest "409::*::*" /ns-${RUNKEY}/foo2/obj1
 
-# test chunk upload
-cat > ${TEST_DATA} <<EOF
-{"chunk_bytes": 1024,
- "total_bytes": ${script_size},
- "content_type": "application/x-bash",
- "content_md5": "$md5"}
+total_bytes=${script_size}
+chunk_bytes=1024
+# test chunk upload (S3 requires at least 5MB chunks)
+if [[ $DEPLOYMENT == "amazons3" ]]
+then
+    chunk_bytes=5242881
+    # generate 5MB + file
+    dd if=/dev/urandom bs=${chunk_bytes} count=1 2>/dev/null | base64 > /tmp/dummy-${RUNKEY}
+    md5=$(mymd5sum < /tmp/dummy-${RUNKEY})
+    total_bytes=$(stat -c "%s" /tmp/dummy-${RUNKEY})
```

Comment:
I think we need an else clause here to copy the script $0 into the dummy file since your code below tries to split the dummy file unconditionally